sdk/python/foundation-models/system/finetune/image-classification/multilabel-classification/hftransformers-fridgeobjects-multilabel-classification.ipynb

```diff
@@ -1019,8 +1019,7 @@
     "\n",
     "request_json = {\n",
     "    \"input_data\": {\n",
-    "        \"columns\": [\"image\"],\n",
-    "        \"data\": [base64.encodebytes(read_image(sample_image)).decode(\"utf-8\")],\n",
+    "        \"data\": [base64.b64encode(read_image(sample_image)).decode(\"utf-8\")],\n",
```

Comment:
can you change the request format as discussed?

Reply:
Yes modified changes as discussed